tests/integrated/test-dataiterator/runtest

```diff
@@ -17,9 +17,6 @@ MPIRUN=getmpirun()
 
 print("Making DataIterator test")
 s, out = launch_safe("make > make.log",pipe=True)
```

Comment:
Note that `launch _safe` launches an mpi job with the max number of procs (if nproc is not given). I think you want `shell_safe` here.

Reply:
fixed 👍